Functions/ImageProcessing/xASL_im_ASLSubtractionAveraging.m

```diff
@@ -257,7 +261,7 @@
 
     %% 2C. Non-time encoded subtraction
     else
-        if nVolumes>1 && ~x.modules.asl.bContainsDeltaM
+        if nVolumes>1 && ~x.modules.asl.bContainsDeltaM && ~x.modules.asl.bContainsCBF 
```

Comment:
Why do we need this new Boolean? We have always used the ApplyQuantification booleans for this, and called this bContainsDeltaM anyway right.

Reply:
OK, another way to clarify this would be calling it `bContains_DeltaM_CBF`, because PWI might be confusing with PWI4D etc.

Reply:
OK how about bContainsSubtracted? If you agree, you can do this, I won't change my mind :)